packages/maskbook/src/plugins/FileService/database.ts

```diff
@@ -1,12 +1,12 @@
 import { createPluginDatabase } from '../../database/Plugin/wrap-plugin-database'
```

Comment:
This file should be in ./Worker because it's only started in the Worker entry of this plugin.